addons/stock/models/stock_picking.py

```diff
@@ -469,6 +469,12 @@ def _compute_has_tracking(self):
         for picking in self:
             picking.has_tracking = any(m.has_tracking != 'none' for m in picking.move_ids)
 
+    @api.depends('move_ids')
```

Comment:
`move_ids.group_id` to avoid retriggering it too much

Reply:
done thanks <3